config/prometheus/rbac.yaml

```diff
@@ -0,0 +1,31 @@
+# creates Role and RoleBinding for prometheus-k8s service account to access our namespace
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: prometheus-k8s
+  namespace: system
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - endpoints
+  - pods
+  verbs:
+  - get
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: prometheus-k8s
+  namespace: system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: prometheus-k8s
+subjects:
+- kind: ServiceAccount
+  name: prometheus-k8s
+  namespace: openshift-monitoring
```

Comment:
This is what is instructed in this [article](https://rhobs-handbook.netlify.app/products/openshiftmonitoring/collecting_metrics.md/).
If already exists,  no change will be reflected.

Reply:
again, likely silly but please let's rename all generic names to be prefixed with `nrop-` or anything. And yes, we have existing and larger issues, but we should start going in the right direction rather than hoping for non-worse. I think the rename should be little effort so let's just do this and we can merge. (same goes for the RTE PR)